Sources/GRPCCore/Documentation.docc/Documentation.md

```diff
@@ -7,17 +7,36 @@ A gRPC library for Swift written natively in Swift.
 
 ## Package structure
 
-gRPC Swift is made up of a number of modules, each of which is documented separately. However this
-module – ``GRPCCore`` – includes higher level documentation such as tutorials. The following list
-contains products of this package:
-
-- ``GRPCCore`` contains core types and abstractions and is the 'base' module for the project.
-- `GRPCInProcessTransport` contains an implementation of an in-process transport.
-- `GRPCHTTP2TransportNIOPosix` provides client and server implementations of HTTP/2 transports built
-  on top of SwiftNIO's POSIX Sockets abstractions.
-- `GRPCHTTP2TransportNIOTransportServices` provides client and server implementations of HTTP/2
-  transports built on top of SwiftNIO's Network.framework abstraction, `NIOTransportServices`.
-- `GRPCProtobuf` provides serialization and deserialization components for `SwiftProtobuf`.
+gRPC Swift is distributed across multiple Swift packages these are:
+
+- `grpc-swift` (this package) containing core gRPC abstractions and an in-process transport.
+  - GitHub repository: [`grpc/grpc-swift`](https://github.com/grpc/grpc-swift)
+  - Documentation: hosted on the [Swift Package
+    Index](https://swiftpackageindex.com/grpc/grpc-swift/documentation)
+- `grpc-swift-nio-transport` contains high-performance HTTP/2 transports built on top of SwiftNIO.
```

Comment:
Should we make `SwiftNIO` a link to the NIO repo or its docs?

Reply:
Yup, good idea, done.